app/api/event_types.py

```diff
@@ -40,7 +40,7 @@ def before_get_object(self, view_kwargs):
             else:
                 view_kwargs['id'] = None
 
-    decorators = (api.has_permission('is_admin', methods="PATCH,DELETE"),)
+    decorators = (api.has_permission('is_admin', methods="PATCH"),)
```

Comment:
Removing `DELETE` from here doesn't mean that this endpoint won't support deletion. Removing it from decorators means that we won't check for admin access when deleting them which is even worse. Just for information although I don't think this PR is needed.